Documentation/faq.md

```diff
@@ -10,6 +16,24 @@
 
 ### Deployment
 
+#### System requirements
+
+Since etcd writes data to disk, SSD is highly recommended. Default storage size limit is 2GB, configurable up to 8GB. 2GB is a reasonable limit for most use cases. And allocate as much heap size to avoid disk swap. If the etcd storage quota is 2GB, the machine should have at least 2GB of RAM. At CoreOS, etcd cluster is usually deployed on dedicated CoreOS Container Linux machines, with dual-core processors, 2GB of RAM, and 80GB of SSD *at the very least*. **It highly varies on workloads; please test before production deployment**.
+
+Most stable production environment is Linux operating system with amd64 architecture.
```

Comment:
can this link to op-guide/supported-platform instead of copying everything? otherwise the two will get out of sync over time